bibliopixel/remote/control.py

```diff
@@ -34,10 +34,20 @@ def __init__(self, layout, animations,
                  external_access=False, port=5000,
                  title='BiblioPixel Remote', bgcolor='black',
                  font_color='white', default=None,
-                 triggers=[]):
+                 triggers=[], auto_demo=None):
 
         self.internal_delay = 0  # never wait
 
+        if auto_demo:
```

Comment:
Uhhh... Auto demo is specific to remote. Remote sees it and creates a Sequence. Making it an entirely new class would overly complicate it (which you totally *never* do). How the heck would a new class know what to get from RemoteControl.